pino.js

```diff
@@ -35,10 +37,27 @@ function pino (opts, stream) {
   var level = opts.level || 'info'
   var serializers = opts.serializers || {}
   var end = ',"v":' + LOG_VERSION + '}\n'
-  return new Pino(level, stream, serializers, stringify, end, name, hostname, slowtime, '')
+  var cache = opts.extreme ? 4096 : 0
+
+  var logger = new Pino(level, stream, serializers, stringify, end, name, hostname, slowtime, '', cache)
+
+  if (cache) {
+    onExit(function () {
+      if (stream === process.stdout) {
+        console.log(logger.buf)
+      } else if (stream === process.stderr) {
+        console.error(logger.buf)
+      } else {
+        stream.write(logger.buf)
+      }
+      process.exit()
```

Comment:
no you're right, all we have to do is remove process.exit and it should be fine, sorry that shouldn't be there